projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -652,7 +652,7 @@ export class IgxDateRangePickerComponent extends PickersBaseDirective
     /** @hidden @internal */
     public ngOnChanges(changes: SimpleChanges): void {
         if (changes['locale']) {
-            this.inputFormat = DatePickerUtil.getDefaultInputFormat(this.locale || 'en') || DatePickerUtil.DEFAULT_INPUT_FORMAT;
+            this.inputFormat = DateTimeUtil.getDefaultInputFormat(this.locale) || DateTimeUtil.DEFAULT_INPUT_FORMAT;
```

Comment:
Doesn't look like getDefaultInputFormat would return a falsy value in any case, so not sure about the fallback to the default input. Also the util falls back to a different default format for some reason?
Also also (!) - setting the input format public prop like this can be destructive if it's already specified by the user.